CHANGELOG.rst

```diff
@@ -1,5 +1,73 @@
 .. _changelog:
 
+0.7.3
+-----
+
+This release contains some very nice improvements to the file and datasets management and visualization as well as project migrations.
+
+New features
+~~~~~~~~~~~~
+
+- **Dataset removal** A dataset can be now removed from a project from the UI. Note that if a removed dataset was part of a project with the Knowledge Graph activated, its metadata will not be removed.
+
+- **Pinned environments image** An image tag can be specified for launching environments. This can be done in the `renku configuration file <https://renku.readthedocs.io/en/latest/user/templates.html?highlight=.dockerignore#renku>`_.
+
+- **Notifications** A notification system has been added so that users get notified for updates and events related to their projects.
+
+- **Project migration** An automatic migration to a new Renku version is possible. Information about the latest and current Renku CLI and template versions are displayed in the ``Status`` section of a project. Additionally, a ``migrationscheck`` command is available in Renku CLI.
+
+Improvements
+~~~~~~~~~~~~
+
+- **Dataset upload** When uploading files, a progress bar is displayed.
+
+- **Dataset visualization** When listing datasets, the description and author's list of a dataset are cropped. Full description and author list are still available on mouse hovering.
```

Comment:
For a better experience on dataset listing inside projects, description and authors have been cropped. Full description and authors are available when accessing the dataset.